application/tests/librarian/config_loader_test.py

```
@@ -0,0 +1,83 @@
+            cfg = load_config()
+        self.assertEqual(cfg.crossencoder_model, "cross-encoder/ms-marco-MiniLM-L-6-v2")
+        self.assertEqual(cfg.top_k_retrieval, 20)
+        self.assertEqual(cfg.top_k_rerank, 5)
+        self.assertEqual(cfg.link_threshold, 0.8)
+        self.assertEqual(cfg.batch_size, 32)
+        self.assertEqual(cfg.ece_target, 0.10)
+        self.assertEqual(cfg.conformal_alpha, 0.10)
+
+    def test_config_is_frozen(self):
+        cfg = load_config()
+        with self.assertRaises(Exception):
```

Ruff (0.15.17)

[warning] Do not assert blind exception: `Exception`

(B017)